solr/core/src/test-files/solr/collection1/conf/solrconfig-parsing-update-processor-chains.xml

```diff
@@ -109,6 +109,21 @@
     </processor>
   </updateRequestProcessorChain>
 
+  <updateRequestProcessorChain name="parse-date-extraction-util-formats">
+    <processor class="solr.ParseDateFieldUpdateProcessorFactory">
+      <str name="defaultTimeZone">UTC</str>
+      <str name="locale">en</str>
+      <arr name="format">
+        <str>yyyy-MM-dd['T'[HH:mm:ss['.'SSS]['Z'</str>
```

Comment:
After working with you on the previous issue, I now know that `'Z'` isn't appropriate if the defaultTimeZone is configurable, since as-such it's a literal that is ignored -- and the defaultTimeZone would take effect which might not be UTC.  We know 'z' is right now.